src/node.cc

```diff
@@ -1724,14 +1742,18 @@ void SetupProcessObject(Environment* env,
   Local<Object> versions = Object::New(env->isolate());
   READONLY_PROPERTY(process, "versions", versions);
 
-  const char http_parser_version[] = NODE_STRINGIFY(HTTP_PARSER_VERSION_MAJOR)
-                                     "."
-                                     NODE_STRINGIFY(HTTP_PARSER_VERSION_MINOR)
-                                     "."
-                                     NODE_STRINGIFY(HTTP_PARSER_VERSION_PATCH);
+#ifdef NODE_EXPERIMENTAL_HTTP
+  READONLY_PROPERTY(versions,
+                    "llhttp",
+                    FIXED_ONE_BYTE_STRING(env->isolate(), llhttp_version));
+  READONLY_PROPERTY(versions, "http_parser", Null(env->isolate()));
+#else  /* !NODE_EXPERIMENTAL_HTTP */
+  READONLY_PROPERTY(versions, "llhttp", Null(env->isolate()));
```

Comment:
I'm not sure whether we want this behaviour while experimental. Expanding the shape of `process.versions` while not compiled with the experimental flag is surprising behaviour IMO, it should just be completely hidden if you don't enable it.

Reply:
I made a decision to define just either of them.